kubernetes-model/pkg/schemagen/generate.go

```diff
@@ -30,29 +30,36 @@ type PackageDescriptor struct {
 }
 
 type schemaGenerator struct {
-	types              map[reflect.Type]*JSONObjectDescriptor
-	typeNames          map[reflect.Type]string
-	customTypeNames    map[string]string
-	packages           map[string]PackageDescriptor
-	typeMap            map[reflect.Type]reflect.Type
+	types           map[reflect.Type]*JSONObjectDescriptor
+	typeNames       map[reflect.Type]string
+	customTypeNames map[string]string
+	packages        map[string]PackageDescriptor
+	typeMap         map[reflect.Type]reflect.Type
 }
 
-func GenerateSchema(t reflect.Type, packages []PackageDescriptor, typeMap map[reflect.Type]reflect.Type, customTypeNames  map[string]string, moduleName string) (*JSONSchema, error) {
+type CrdScope int32
+
+const (
+	Namespaced CrdScope = 0
+	Cluster    CrdScope = 1
```

Comment:
I see that you're trying to create an enumeration here. I suggest using [iota](https://golang.org/ref/spec#Iota)

```suggestion
	Namespaced CrdScope = iota
	Cluster    CrdScope = iota
```